openweathermap-ios/openweathermap-ios/Maps/ViewControllers/MapViewController.swift

```diff
@@ -48,8 +56,34 @@ extension MapViewController: MKMapViewDelegate {
         return renderer
     }
     
-    func mapViewDidChangeVisibleRegion(_ mapView: MKMapView) {
+    func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
         updateAnnotation(location: mapView.centerCoordinate)
```

Comment:
If you want to have a centralized pin, why not insert a view above the MapView instead of removing/inserting an annotation every time the center coordinates changes?

Reply:
The idea of adding a note in the center was to have the same or a similar iteration as was implemented in the Android app.
As I suggested it has a higher computational cost.
I hadn't thought how to do it any other way.
I liked the suggestion. Thanks.
**Is your suggestion to add an annotation but in an upper layer?**

Reply:
No, I suggested adding a Subview above the MapView. But you made a very good point with the interaction expectation. So, let it go. ❄️ ⛄️